src/contributor.js

```diff
@@ -258,7 +258,7 @@ let contributors = [
     id: 26,
     name: "Leo Cuero",
     cohort: "2023 Alpha",
-    bio: "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.",
+    bio: "My name is Leopoldo F. Cuero, I am from Cali, Colombia. I served for 2 decades in the U.S. Government in multiple roles, I consider myself an unwritten code with a code in mine The more I LEARN the more I realize I need to LEARN more.",
```

Comment:
Generally numbers less than 10 are spelled out, so I would recommend "two decades"
Also I am a little confused about this line: "consider myself an unwritten code with a code in mine The more"